prody/proteins/interactions.py

```diff
@@ -918,29 +957,52 @@ def calcSaltBridgesDCD(atoms, trajectory, distA=4.5):
         except TypeError:
             raise TypeError('coords must be an object '
                             'with `getCoords` method')
+    SBs_all = []
 
-    if isinstance(trajectory, Atomic):
-        trajectory = Ensemble(trajectory)        
+    if trajectory is not None:
+        if isinstance(trajectory, Atomic):
+            trajectory = Ensemble(trajectory)        
                         
-    SBs_all = []
+        trajectory.reset()
```

Comment:
maybe store the current active frame and set it back to there at the end

Reply:
The way to do this would be to have the following line before the reset
```
nfi = trajectory._nfi # next frame index
```
and the following one at the end of the function before the return
```
trajectory._nfi = nfi
```

Reply:
I'd probably also provide a kwarg for whether the user wants to reset back to the beginning. They may want to analyse from frame 3 onwards for example to skip some equilibration period.

Reply:
Done. Also, I added start_frame and stop_frame. Now everyone may choose which frame or model in multi-model PDB analyze.

Reply:
sounds good!